### ISSUE TITLE
Home Tab not accessible or freezing randomly

### ISSUE THREAD

**Issue by abhiTronix** [squidfunk/mkdocs-material#4083]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [ ] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

Hi, I'm seeing this weird behavior that [`Home`](https://abhitronix.github.io/vidgear/v0.2.6-dev/) Tab becoming inaccessible or frozen/disabled _(cannot be clicked)_ when I'm on any another tab such as [`Gear`](https://abhitronix.github.io/vidgear/v0.2.6-dev/gears/) _(Make sure to follow Steps to reproduce)_. It happens randomly but quit frequently and I tried switching browsers and the problem persists on all of them. I have no clue about what is causing this, and nor mkdocs logs or browser inspection mode is showing any errors/issues. 

Moreover, this is happening both on local build as well as when deployed to gh-pages. Also, only `Home` tab is affected and other tabs are accessible normally.

However, this issue wasn't there earlier with same `mkdocs.yml` configuration but with MkDocs 1.2.3 and mike 1.1.2 (Test here https://abhitronix.github.io/vidgear/v0.2.5-stable/)

### Expected behaviour

I should able to click [`Home`](https://abhitronix.github.io/vidgear/v0.2.6-dev/) Tab when I'm on any another tab such as [`Gear`](https://abhitronix.github.io/vidgear/v0.2.6-dev/gears/)

### Actual behaviour

 [`Home`](https://abhitronix.github.io/vidgear/v0.2.6-dev/) Tab is inaccessible or frozen/disabled _(cannot be clicked)_ when I'm on any another tab such as [`Gear`](https://abhitronix.github.io/vidgear/v0.2.6-dev/gears/) _(Make sure to follow Steps to reproduce)_

### Steps to reproduce

1. Go to [`Home`](https://abhitronix.github.io/vidgear/v0.2.6-dev/) Tab
2. Go to any another tab such as [`Gear`](https://abhitronix.github.io/vidgear/v0.2.6-dev/gears/)
3. Try clicking `Home` tab is now impossible.

### Package versions

- Python: 3.7
- MkDocs: MkDocs 1.3.0 and mike 1.1.2
- Material: 8.3.8

### Configuration

```yaml
site_name: VidGear
site_url: https://abhitronix.github.io/vidgear/
```
Complete `mkdocs.yml`: https://github.com/abhiTronix/vidgear/blob/testing/mkdocs.yml

### System information

- Operating system: Windows
- Browser: Chrome, Edge, Mozilla

**Comment by squidfunk** [comment]
Thanks for reporting! I can reproduce the error on your documentation, and I've stepped through the debugger on your page. I think I've found the error and fixed it in a26113020, so please re-deploy with `master` to check if it's gone.

---

Other than that, the next time you file a report, please make sure you can check all boxes, i.e. check whether the problem persists when you remove all customizations. Customizations are more than often the reason for errors, and I'm afraid I can't help everybody to customize his/her documentation to taste. Also, debugging on a live page is likely not to return any meaningful results. The time I spend trying to come up with reproductions is often better invested in new features 😉

I understand that creating reproductions sometimes involves serious effort, but it allows me to provide fixes almost instantly. The issue reporting process will shortly be transformed to require reproducible examples.

**Comment by abhiTronix** [comment]
> Thanks for reporting! I can reproduce the error on your documentation, and I've stepped through the debugger on your page. I think I've found the error and fixed it in https://github.com/squidfunk/mkdocs-material/commit/a261130204371e7dab3d4a961926b2cde43048b4, so please re-deploy with master to check if it's gone.

@squidfunk Thank you for the quick fix. Sorry I was traveling and did not have access to my laptop. But now I'll test this fix right away and let you know.

> Other than that, the next time you file a report, please make sure you can check all boxes, i.e. check whether the problem persists when you remove all customizations. Customizations are more than often the reason for errors, and I'm afraid I can't help everybody to customize his/her documentation to taste. Also, debugging on a live page is likely not to return any meaningful results. The time I spend trying to come up with reproductions is often better invested in new features 😉

Sorry, I totally understand your point but I didn't got the chance to test if the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`. I thought I'll check the box once I'm sure the custom directory isn't the problem. The sudden travelling plan got in the way and I forgot about it untill now. I'll take care of this next time. 😃

**Comment by abhiTronix** [comment]
@squidfunk Unforuntely I'm still seeing this bug with `Home` tab 

I turn on the debugger and checked JS files and I think problem is now with `bundle.0238f547.min.js.map`(showing 404 error):

![image](https://user-images.githubusercontent.com/34266896/177039960-938b6967-903a-46b6-b6c8-2a0422b9d099.png)

You can test new deployed version here: https://abhitronix.github.io/vidgear/v0.2.6-dev/ the problem still persists.

**Comment by squidfunk** [comment]
Works for me:

https://user-images.githubusercontent.com/932156/177040308-367782aa-63e6-4d44-a90d-115bd9a49f4e.mp4

The error you're reporting may be related to an incomplete deployment, i.e. assets have not yet been propagated to the caches. Please make sure that the application bundle loads properly in your browser...

**Comment by abhiTronix** [comment]
@squidfunk Sorry I think I said it too early the issue is fixed now. I pressed `Ctrl+F5` and problem was gone. I think my browser cached JS files. Thank you so much. Love your work!!! ❤️

**Comment by squidfunk** [comment]
Released as part of 8.3.9.
